src/index.html

```diff
@@ -6,13 +6,51 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
+
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <title>Product cards</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="./styles/index.css"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__image"></div>
+      <div class="card__text">
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__description">Product code: 195434</p>
+      </div>
+      <div class="card__review">
+        <div class="card__stars">
```

Comment:
Here you should have used the same Block from the *Stars* task.
The one with modifiers for how many stars should be active or not.

Reply:
Hey pedro-ruas, thank you for your feedback. I had been fixing this to implement in next activity as well. Now i think that everything is fine.